test/agentchat/contrib/test_pgvector_retrievechat.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3 -m pytest
+
+import os
+import sys
+
+import pytest
+from sentence_transformers import SentenceTransformer
+
+from autogen import config_list_from_json
+from autogen.agentchat.contrib.retrieve_assistant_agent import RetrieveAssistantAgent
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+try:
+    import pgvector
+
+    from autogen.agentchat.contrib.retrieve_assistant_agent import (
+        RetrieveAssistantAgent,
+    )
+    from autogen.agentchat.contrib.retrieve_user_proxy_agent import (
+        RetrieveUserProxyAgent,
+    )
+except ImportError:
+    skip = True
+else:
+    skip = False
+
+
+test_dir = os.path.join(os.path.dirname(__file__), "../..", "test_files")
+
+
+@pytest.mark.skipif(
+    sys.platform in ["darwin", "win32"] or skip,
```

Comment:
This test needs to be skipped also when skip_openai is True. No need to skip by platform.

Reply:
I have made the corrections @sonichi 

Please let me know if there are further corrections needed.